easybuild/easyconfigs/m/MEGAHIT/MEGAHIT-1.1.2-foss-2018a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'MakeCp'
+
+name = 'MEGAHIT'
+version = '1.1.2'
```

Comment:
If `Python` is a dependency, we usually make that clear via the `versionsuffix` (since `Python` is a very common dependency)

```python
versionsuffix = '-Python-%(pyver)s`
```

Note that including this implies that you need to rename the easyconfig file to end with `-Python-2.7.14.eb`...